src/util.js

```diff
@@ -363,8 +363,13 @@ define([
         ], lang);
     };
 
-    that.isCaseReference = function (value) {
-        return value.startsWith('#case/');
+    that.getReferenceName = function (value) {
+        var ref = /^#([^\/]+)\//.exec(value);
+        if (!ref) {
+            return "Form Reference";
+        }
+        ref = ref[1][0].toUpperCase() + ref[1].substring(1).toLowerCase();
+        return ref + " Reference";
```

Comment:
This is used for analytics. It returns "Case Reference" and "Form Reference" (the same categories that have been used for hashtag analytics up to this point), and will also send a new category for each new hashtag namespace, for exmaple "User Reference" for `#user/...` hashtags. FYI @dimagi/product